config/software/chef.rb

```diff
@@ -63,6 +63,8 @@
   excluded_groups << "ruby_prof" if aix?
   excluded_groups << "ruby_shadow" if aix?
 
+  bundle "update --bundler"
```

Comment:
@ilancb4  - i don't think you have to update.
Remove Gemfile.lock and update bundle on build machine should do the job

Reply:
Without updating the build somehow thinks that the bundler isn't installed so I suggest we leave it for safety.

Reply:
Thanks @yanivmn we tried that and have already updated the dependency outside the build (the environment dependency). 
All this change does it to prevent us from falling on an error saying that bundler 1.17.3 is not installed